### PR TITLE
[14.0][MIG] stock_product_qty_by_packaging: Migration to 14.0

### PR DESCRIPTION
**Pull request by hiepnh97**
Notes:
- Depend on: [PR 1245](https://github.com/OCA/stock-logistics-warehouse/pull/1245)
- Pre-commit failed because I want to disable the conflict module.
- I will delete `test-requirement.txt` when depending [PR 1245](https://github.com/OCA/stock-logistics-warehouse/pull/1245) was Merge.

### CODE REVIEW

**Review by simahawk — COMMENTED** [review]
LG waiting for the dep to get merged

**Comment by simahawk** [comment]
can you check runbot?

**Comment by simahawk** [comment]
pls rebase instead of merging

**Review by TDu — APPROVED** [review]

**Review by simahawk — CHANGES_REQUESTED** [review]
the PR is about "stock_product_qty_by_packaging" but the code is about "stock_packaging_calculator_packaging_type". Messed up branches? :smile:

**Comment by hiepnh97** [comment]
> the PR is about "stock_product_qty_by_packaging" but the code is about "stock_packaging_calculator_packaging_type". Messed up branches? 😄

Sorry, I have the wrong step when rebase and re-push 😄Please help help re-check, Thanks

File: stock_product_qty_by_packaging/models/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Camptocamp SA
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# @author: Sébastien Alix <sebastien.alix@camptocamp.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
```
Comment (simahawk) [inline stock_product_qty_by_packaging/models/stock_move.py]: the license is mixed, probably my fault: could you replace "LGPL" everywhere w/ "AGPL"?

**Review by simahawk — CHANGES_REQUESTED** [review]

File: stock_product_qty_by_packaging/models/stock_move.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Camptocamp SA
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# @author: Sébastien Alix <sebastien.alix@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/AGPL)
```
Comment (simahawk) [inline stock_product_qty_by_packaging/models/stock_move.py]: sorry but links are lowercase otherwise they are broken

**Review by simahawk — APPROVED** [review]

**Pull request merged**